### DISCUSSION TITLE
subset data

### DISCUSSION THREAD

**Discussion by emihoe** [cafferychen777/ggpicrust2#46] [Q&A] [answered]
Hello,
I would like to subset my picrust2 data once I've integrated my abundance table and metadata. I would like to subset from one of my metadata columns (i.e. subset based on soil sample, excluding all "water" samples). 
How can I do this?
Thank you

**Comment by cafferychen777** [comment, marked as answer]
Hello @emihoe ,

You can use the following code similar.

```
sub_metadata <- metadata %>% filter(your_group_column != "water")
sub_abundance <- abundance[, sub_metadata$your_sample_id_column]
```

Best regards,
